index.html

```diff
@@ -13,7 +13,7 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>My Portfolio</title>
+  <title>Nizamuddin</title>
```

Comment:
- [ ] Kindly update your title to it adequately and briefly describes the content of the page, example:
`Nizamuddin's portifolio`
`Nizamuddin full-stack web developer`